### PR TITLE
add ramda

### PR DESCRIPTION
**Pull request by jumblerg**

### CODE REVIEW

File: ramda/build.boot [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+(def +lib-version+ "0.22.1")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/ramda
+       :version     +version+
+       :description "A practical functional library for Javascript programmers."
+       :url         "http://ramdajs.com/"
+       :scm         {:url "https://github.com/ramda/ramda"}
+       :license     {"CC BY-NC-SA 3.0" "https://creativecommons.org/licenses/by-nc-sa/3.0"}})
```
Comment (miikka) [inline ramda/build.boot]: Isn't [Ramda license](https://github.com/ramda/ramda/blob/master/LICENSE.txt) MIT?

Comment (Deraen) [inline ramda/build.boot]: Yes: https://github.com/ramda/ramda/blob/master/LICENSE.txt#L1

**Comment by jumblerg** [comment]
fixed

**Comment by Deraen** [comment]
Something is wrong with the build: `browserify` can't be found.

**Comment by Deraen** [comment]
Okay, it is the sdp-transform package here that is breaking the build. I separated Ramda commits to PR #867 and merged that.

@jumblerg You'll need to create another PR for sdp-transform package.

**Pull request closed without merging**